README.md

```diff
@@ -222,6 +222,8 @@ You can also [generate global inputs using Python (see python/global_data_proces
 | `crop_set`      | user-set real value of constant crop vegetation type heights (m) (only used if `crop_opt=1`) |
 | `co2_opt`       | user-set options for applying a CO2 inhibition factor for biogenic isoprene-only emissions using either the [Possell & Hewitt (2011)](https://doi.org/10.1111/j.1365-2486.2010.02306.x) (= `0`, default) or [Wilkinson et al. (2009)](https://doi.org/10.1111/j.1365-2486.2008.01803.x) method (= `1`). Use of option = `1` (Possell & Hewitt 2011) is especially recommended for sub-ambient CO2 concentrations.  To turn off co2 inhibition factor set `co2_opt=2`  |
 | `co2_set`       | user-set real value of atmospheric co2 concentration (ppmv) (only used if `co2_opt=0` or `co2_opt=1`) |
+| `leafage_opt`   | user-set options for applying leaf-age response to biogenic VOC emissions based on [Guenther et al. 2006](https://doi.org/10.5194/acp-6-3181-2006) (= `0`, default, otherwise, e.g. `leafage_opt=1`, the corresponding $\gamma$ is set to 1). |
+| `tsteplaiinput` | user-defned options for number of seconds in the interval at which LAI (Leaf Area Index) data is provided to the model. For instance, if LAI data is given on a daily basis, tsteplaiinput would be set to the number of seconds in a day (86,400 seconds). If LAI data is provided monthly, then tsteplaiinput would represent the total number of seconds in that month (e.g., 2,592,000 seconds for a 30-day month). This parameter helps in determining the frequency of LAI input and is crucial for interpolating LAI values to the model's hourly timesteps when the model's timestep (time_intvl) is smaller than the LAI input interval.  |
```

Comment:
Yes, it will be hard to get daily CLU since we only have climatological (2001-2017) monthly values. Daily linear interpolation for LAI is probably the best we can get.